charts/spire-client-example/README.md

```diff
@@ -1,7 +1,5 @@
 # spire-client-example
 
-<!-- This README.md is generated. -->
```

Comment:
Please us the template in the command.

https://github.com/philips-labs/helm-charts/blob/main/.github/README.md.tmpl

```shell
helm-docs -g charts/spire -t .github/README.md.tmpl
```

I should probably document this in a CONTRIBUTING.md